posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<p>"We save this with the file name <code>lib_interp.f90</code> and then call the command:"</p><p>"We save this with the file name <code>lib_interp.f90</code> and then compile it on a terminal with the command:"</p><p>Of something like that to highlight that what you're doing is compiling the program and that you can't run that in a notebook, at least not in the way is written</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>